### PR TITLE
JavaScript How This Course Will Work: Remove duplication

### PR DESCRIPTION
**Pull request by nikitarevenco**
## Because

In the ruby section there is duplication about the last project in the course, this paragraph already exists and it is relevant regardless of the track, so it makes no sense to include it in the lesson.

## This PR

- Remove unnecessary paragraph

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: javascript/introduction/how_this_course_will_work.md [hunk 1]
```diff
@@ -11,9 +11,10 @@ How is this course set up? It depends a bit on the path you have selected:
 <details markdown="block">
   <summary class="dropDown-header">Ruby Track</summary>
 
-It starts with a deeper look at the basics, just like the [Ruby Programming course](/paths/full-stack-ruby-on-rails/courses/ruby-programming) did with Ruby. You don't need to have completed the Ruby Programming course or [the Ruby on Rails course](/paths/full-stack-ruby-on-rails/courses/ruby-on-rails) to understand these basics. We won't be focusing deeply on the really basic coding items, so it will move quickly. You should, however, already have completed the [Foundations course](https://www.theodinproject.com/paths/foundations/courses/foundations) -- specifically, the [JavaScript Basics section](https://www.theodinproject.com/paths/foundations/courses/foundations#javascript-basics) -- before starting this course.
+It starts with a deeper look at the basics, just like the [Ruby Programming course](/paths/full-stack-ruby-on-rails/courses/ruby-programming) did with Ruby. You don't need to have completed the Ruby Programming course or [the Ruby on Rails course](/paths/full-stack-ruby-on-rails/courses/ruby-on-rails) to understand these basics.
```
Comment (JoshDevHub) [inline javascript/introduction/how_this_course_will_work.md:14]:
```suggestion
It starts with a deeper look at the basics, just like the [Ruby Programming course](/paths/full-stack-ruby-on-rails/courses/ruby-programming) did with Ruby. You don't need to have completed the Ruby Programming course or [the Ruby on Rails course](/paths/full-stack-ruby-on-rails/courses/ruby-on-rails) to understand these basics.
```
Trailing space

**Review by JoshDevHub — CHANGES_REQUESTED** [review]
Ah nice. I missed this duplication.

One small fix and I'll merge.

**Review by JoshDevHub — APPROVED** [review]

**Comment by fabulousgk** [comment]
This actually is not quite reight either as the two tracks have repeated info, and the paragraph moved applies mostly to the ruby track not JS (ie the way it is worded), not the JS section also mentions a final project.

I will work to refine it more, since this was my change to begin with.

**Comment by nikitarevenco** [comment]
> This actually is not quite reight either as the two tracks have repeated info, and the paragraph moved applies mostly to the ruby track not JS (ie the way it is worded), not the JS section also mentions a final project.
> 
> I will work to refine it more, since this was my change to begin with.

Okay. Good luck!👍

**Pull request merged**